mitodl/micromasters
# Profiles creation

#### What are the relevant tickets?
fixes #73 

#### What's this PR do?
Creates a Profile for the user and populates it with the data existing in the connected EDX instance

#### Where should the reviewer start?
`pipeline_api.py`

#### How should this be manually tested?
Add some personal info for your user on edx and then login again on micromasters.
The user should be new on micromasters otherwise the profile will not be updated with the info from edx

#### Any background context you want to provide?
you need a working micromasters properly configured with a local instance of EDX

#### What GIF best describes this PR or how it makes you feel?
![](http://2.bp.blogspot.com/-B6-lf4Ggr1I/U878LbKyJnI/AAAAAAAAJkQ/W3xaWSrtIDQ/s1600/nsa.gif)
